lib/CustomFields/pages/EditCustomFieldsSettings/EditCustomFieldsSettings.js

```diff
@@ -143,17 +199,26 @@ const EditCustomFieldsSettings = ({
         ? (
           permissions.canEdit
             ? (
-              <CustomFieldsForm
-                entityType={entityType}
-                initialValues={getInitialValues()}
-                redirectToView={redirectToView}
-                onSubmit={saveCustomFields}
-                submitting={submitting}
-                submitSucceed={submitFinished}
-                fetchUsageStatistics={fetchUsageStatistics}
-                sectionTitle={sectionTitle.value || defaultSectionTitle}
-                permissions={permissions}
-              />
+              <>
```

Comment:
I don't think that we need extra `<>` here because there's only one component returned

Reply:
@BogdanDenis, good catch, thank you!